lib/galaxy/util/__init__.py

```diff
@@ -1069,7 +1069,7 @@ def listify(item: Any, do_strip: bool = False) -> List:
     :rtype:             list
     :returns:           The input as a list
     """
-    if not item:
+    if item is None or item == "":
```

Comment:
This seems like a pretty risky change. Can you move the special casing to munge_filters ? It does change things i.e. when item is 0.